knpy/data_utils.py

```diff
@@ -1,17 +1,19 @@
 import os
+
 BRAID_PATH = "data_knots/prime_knots_in_braid_notation.csv"
 
 current_dir = os.path.dirname(__file__)
 file_path = os.path.join(current_dir, BRAID_PATH)
 
-knots_in_braid_notation_dict = dict()
-with open(file_path, 'r') as file:
-    for i,line in enumerate(file):
-        if i==0: continue
+knots_in_braid_notation_dict = {}
+with open(file_path, "r") as file:
+    for i, line in enumerate(file):
+        if i == 0:
+            continue
         line = line.strip()
         line_splitted = line.split(",")
         key = line_splitted[0]
-        temp_braid_notations = [] #TODO Multiple braid notation might be available
+        temp_braid_notations = []  # TODO Multiple braid notation might be available
```

Comment:
W0511: TODO Multiple braid notation might be available (fixme)

Reply:
Fixme type warnings are ignored in CI, so I left it here. It's not _trivially_ fixable anyways.